src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -61,6 +62,11 @@
     private static final String AUTH_HEAD = "Basic";
 
     /**
+     * Split pattern.
+     */
+    private static final Pattern PATTERN = Pattern.compile(PsBasic.AUTH_HEAD);
```

Comment:
@dalifreire it shouldn't, you use `SPLIT_PATTERN` in other place, which is longer than `AUTH_PATTERN`